public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.ts

```diff
@@ -217,26 +226,43 @@ function getElements(state: DashboardSceneState) {
           spec: {
             pluginVersion: vizPanel.state.pluginVersion ?? '',
             options: vizPanel.state.options,
-            fieldConfig: (vizPanel.state.fieldConfig as FieldConfigSource) ?? defaultFieldConfigSource(),
+            fieldConfig: vizFieldConfig ?? defaultFieldConfigSource(),
           },
         },
       },
     };
-    acc.push(elementSpec);
-    return acc;
-  }, []);
+
+    return elementSpec;
+  });
+
   // create elements
 
   const elements = createElements(panelsArray);
   return elements;
 }
 
 function getPanelLinks(panel: VizPanel): DashboardLink[] {
-  const vizLinks = dashboardSceneGraph.getPanelLinks(panel);
-  if (vizLinks) {
-    return (vizLinks.state.rawLinks as DashboardLink[]) ?? [];
-  }
-  return [];
+  const vizLinks = dashboardSceneGraph.getPanelLinks(panel)?.state?.rawLinks ?? [];
+
+  return vizLinks.map((link) => {
+    const dashLink: DashboardLink = {
+      title: link.title,
+      targetBlank: link.targetBlank ?? false, // TODO: should this default to false?
```

Comment:
`targetBlank` is optional in the source `DataLink`, but is mandatory in the target `DashboardLink`. What should it default to?